[2023-01-04 04:10:15,227] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-02T07:00:00+00:00 [queued]>
[2023-01-04 04:10:15,236] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-02T07:00:00+00:00 [queued]>
[2023-01-04 04:10:15,237] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 04:10:15,237] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-04 04:10:15,238] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 04:10:15,270] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_crypto_values> on 2021-03-02 07:00:00+00:00
[2023-01-04 04:10:15,276] {standard_task_runner.py:52} INFO - Started process 61713 to run task
[2023-01-04 04:10:15,344] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'populate_crypto_values', 'get_crypto_values', 'scheduled__2021-03-02T07:00:00+00:00', '--job-id', '89', '--raw', '--subdir', 'DAGS_FOLDER/populate_crypto_values.py', '--cfg-path', '/tmp/tmpb7w2gf42', '--error-file', '/tmp/tmpjxbjf3re']
[2023-01-04 04:10:15,349] {standard_task_runner.py:80} INFO - Job 89: Subtask get_crypto_values
[2023-01-04 04:10:15,439] {task_command.py:370} INFO - Running <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-02T07:00:00+00:00 [running]> on host xuxon-virtual-machine
[2023-01-04 04:10:15,571] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=populate_crypto_values
AIRFLOW_CTX_TASK_ID=get_crypto_values
AIRFLOW_CTX_EXECUTION_DATE=2021-03-02T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-02T07:00:00+00:00
[2023-01-04 04:10:16,732] {logging_mixin.py:115} INFO - Encountered errors while inserting rows: [{'index': 0, 'errors': [{'reason': 'invalid', 'location': 'snapshot_date', 'debugInfo': '', 'message': "Invalid date: '03-03-2021'"}]}, {'index': 1, 'errors': [{'reason': 'invalid', 'location': 'snapshot_date', 'debugInfo': '', 'message': "Invalid date: '03-03-2021'"}]}]
[2023-01-04 04:10:16,732] {python.py:173} INFO - Done. Returned value was: None
[2023-01-04 04:10:16,796] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=populate_crypto_values, task_id=get_crypto_values, execution_date=20210302T070000, start_date=20230104T071015, end_date=20230104T071016
[2023-01-04 04:10:16,920] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-04 04:10:16,979] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-01-04 05:27:03,284] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-02T07:00:00+00:00 [queued]>
[2023-01-04 05:27:03,300] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-02T07:00:00+00:00 [queued]>
[2023-01-04 05:27:03,300] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 05:27:03,300] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-04 05:27:03,300] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-04 05:27:03,314] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_crypto_values> on 2021-03-02 07:00:00+00:00
[2023-01-04 05:27:03,319] {standard_task_runner.py:52} INFO - Started process 9141 to run task
[2023-01-04 05:27:03,325] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'populate_crypto_values', 'get_crypto_values', 'scheduled__2021-03-02T07:00:00+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/populate_crypto_values.py', '--cfg-path', '/tmp/tmp_sn8v26p', '--error-file', '/tmp/tmp6me6yo3p']
[2023-01-04 05:27:03,326] {standard_task_runner.py:80} INFO - Job 118: Subtask get_crypto_values
[2023-01-04 05:27:03,386] {task_command.py:370} INFO - Running <TaskInstance: populate_crypto_values.get_crypto_values scheduled__2021-03-02T07:00:00+00:00 [running]> on host xuxon-virtual-machine
[2023-01-04 05:27:03,513] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=populate_crypto_values
AIRFLOW_CTX_TASK_ID=get_crypto_values
AIRFLOW_CTX_EXECUTION_DATE=2021-03-02T07:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-02T07:00:00+00:00
[2023-01-04 05:27:05,127] {logging_mixin.py:115} INFO - New rows have been added.
[2023-01-04 05:27:05,128] {python.py:173} INFO - Done. Returned value was: None
[2023-01-04 05:27:05,138] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=populate_crypto_values, task_id=get_crypto_values, execution_date=20210302T070000, start_date=20230104T082703, end_date=20230104T082705
[2023-01-04 05:27:05,168] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-04 05:27:05,189] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
